pages/LearnerDashboard.html

```diff
@@ -21,7 +21,7 @@ <h2>
     Here's how the learner dashboard looks to an experienced learner:
   </p>
 
-  <img src="static/images/activeLearnerDashboard.png" style="width: 100%;">
+  <img src="static/images/activeLearnerDashboard.png" style="width: 100%;" alt="A heading, 'Learner Dashboard' with a blue background, followed by a white panel consisting of 'Completed Explorations', 'Completed Collections' and 'Subscriptions' trackers. A white left-aligned Nav-bar is present below the trackers, where 'in-progress explorations' is highlighted. Three coloured cards are displayed, along with a 'sort-by' dropdown list on the top-right corner. ">
```

Comment:
(You probably don't need to worry too much about describing all the colors in detail.)

Amend intro sentence to: A page with the heading 'Learner Dashboard', followed by a white panel consisting of ...

Not sure what "trackers" means here, please explain in layperson terms.

Next sentence: On the left, below the trackers, there is a menu showing XXX, with YYY highlighted.

Explain what the three coloured cards are -- a screenreader user will have no context based on your description here.

Also, break the last sentence into two; they talk about different things. The last sentence should be: There is a 'sort-by' dropdown at the top-right corner.

In general, please use layperson terminology. Non-developers will not understand terms like "nav bar" and "trackers".